Java: Use List Values with IN (#451)
cap-js/docs: java/query-api.md

For example, the following query returns all sales after Q2/2012:

```java
import static com.sap.cds.ql.CQL.*;

CqnListValue props = list(get("year"), get("quarter"));
CqnListValue vals  = list(val(2012), val(2));
CqnSelect q = Select.from(SALES).where(comparison(props, GT, vals));
```

> You can also compare multiple list values at once in a `IN` predicate - for example to efficiently filter by mutiple key value sets:

Reviewer: ```suggestion
You can also compare multiple list values at once using an `IN` predicate - for example to efficiently filter by multiple key value sets:
```

```java
import static com.sap.cds.ql.CQL.*;

CqnListValue elements = list(get("AirlineID"), get("ConnectionID"));
CqnListValue lh454  = list(val("LH"), val(454));
CqnListValue ba119  = list(val("BA"), val(119));

> List<CqnListValue> valueSets = List.of(lh454, ba119);
> 
> CqnSelect q = Select.from(FLIGHT_CONNECTION).where(in(elements, valueSets));

Reviewer: ```suggestion

CqnSelect q = Select.from(FLIGHT_CONNECTION).where(in(elements, List.of(lh454, ba119)));
```
